Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/IISWebAppDeployV1/DeployIISWebApp.ps1

```diff
@@ -40,6 +41,19 @@ function Get-ScriptToRun
     $websiteName = EscapeSpecialChars -str $websiteName
     $AdditionalArguments = EscapeSpecialChars -str $AdditionalArguments
 
+    $useSanitizerCall = Get-SanitizerCallStatus
+    $useSanitizerActivate = Get-SanitizerActivateStatus
+
+    if ($useSanitizerCall) 
+    {
+        $sanitizedArguments = Protect-ScriptArguments -InputArgs $AdditionalArguments -TaskName "IISWebAppDeployV1"
```

Comment:
If useSanitizerActivate then sanitizedArguments  is not used. Should we fill this only if useSanitizerActivate  == true?